hypervisor/include/arch/x86/asm/tsc.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2018 Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef ARCH_X86_TSC_H
+#define ARCH_X86_TSC_H
+
+#include <types.h>
+
+#define TSC_PER_MS	((uint64_t)get_tsc_khz())
+
+/**
+ * @brief read tsc.
```

Comment:
Can you be a bit more verbose here, i.e.: 
```suggestion
 * @brief Read Time Stamp Counter (TSC)
```